pkg/sql/plan/utils.go

```diff
@@ -1652,3 +1654,50 @@ func ResetAuxIdForExpr(expr *plan.Expr) {
 		}
 	}
 }
+
+func SubstitueParam(expr *plan.Expr, proc *process.Process) *plan.Expr {
+	switch t := expr.Expr.(type) {
+	case *plan.Expr_F:
+		for _, arg := range t.F.Args {
+			SubstitueParam(arg, proc)
+		}
+	case *plan.Expr_P:
```

Comment:
you can deal with plan.Expr_V too
that's expr for @.   and can replace with const value

Reply:
fixed